types/storybook__addon-knobs/index.d.ts

```diff
@@ -56,3 +56,7 @@ export interface WrapStoryProps {
 
 export function withKnobs(storyFn: RenderFunction, context: StoryContext): React.ReactElement<WrapStoryProps>;
```

Comment:
Is this react-specific? Should we avoid re-exporting this from `angular.d.ts` and `vue.d.ts`?

Reply:
I believe because this is the plugin function, the signature is still correct - @ndelangen can you clarify, please?

Reply:
@ndelangen @hypnosphi @shilman could you shed some light on this, please?

Reply:
One should look at implementations to figure out the return types. See vue for instance https://github.com/storybooks/storybook/blob/master/addons/knobs/src/vue/index.js

Reply:
>  so the signature should be framework-dependent

I think this could be done by making a different version of the function in each of `angular.d.ts`, `react.d.ts`, and `vue.d.ts`.